modules/ocf/manifests/extrapackages.pp

```diff
@@ -15,7 +15,6 @@
   include ocf::packages::chrome
   include ocf::packages::emacs
   include ocf::packages::kubectl
-  include ocf::packages::matplotlib
```

Comment:
I don't think this should be removed, otherwise looks great!

Reply:
Should be fixed!